ci/pod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+spec:
+  # XXX: tmp hack to get anyuid SCC; need to ask to get jenkins SA added
+  serviceAccountName: papr
```

Comment:
Fine for now, though if we start spreading this across multiple projects I think this is the kind of thing we'll want to abstract out.